akka-docs/rst/java/stream/stream-quickstart.rst

```diff
@@ -251,7 +251,7 @@ Graphs are constructed using :class:`GraphDSL` like this:
 
 As you can see, we use graph builder ``b`` to construct the graph using ``UniformFanOutShape`` and ``Flow`` s.
 
-``GraphDSL.create`` returns a :class:`Graph`, in this example a ``Graph<ClosedShape,Unit>`` where
+``GraphDSL.create`` returns a :class:`Graph`, in this example a ``Graph<ClosedShape,NotUsed>`` where
```

Comment:
[This](http://doc.akka.io/docs/akka/2.4/java/stream/stream-quickstart.html#Broadcasting_a_stream) refers to [TwitterStreamQuickstartDocTest.java](https://github.com/akka/akka/blob/4a7cd147313404eb8137e56163e54f06c23640d0/akka-docs/rst/java/code/docs/stream/TwitterStreamQuickstartDocTest.java#L290)

```
RunnableGraph.fromGraph(GraphDSL.create(b -> {
```

where [`GraphDSL.create()`](https://github.com/akka/akka/blob/4a7cd147313404eb8137e56163e54f06c23640d0/akka-stream/src/main/boilerplate/akka/stream/javadsl/GraphCreate.scala.template#L15L16) is defined as follows:

```
  def create[S <: Shape](block: function.Function[GraphDSL.Builder[NotUsed], S]): Graph[S, NotUsed] =
    scaladsl.GraphDSL.create() { b ⇒ block.apply(b.asJava) }
```

So `NotUsed` is correct here.